tfe/resource_tfe_registry_module.go

```diff
@@ -67,8 +67,13 @@ func resourceTFERegistryModule() *schema.Resource {
 						},
 						"oauth_token_id": {
 							Type:     schema.TypeString,
-							Required: true,
 							ForceNew: true,
+							Optional: true,
+						},
+						"github_app_installation_id": {
+							Type:     schema.TypeString,
+							ForceNew: true,
+							Optional: true,
```

Comment:
If we're satisfied with the `ConflictsWith` & `AtleastOneOf` schema validations in `tfe/resource_tfe_workspace.go`, add here as well.

```
		"oauth_token_id": {
			Type:          schema.TypeString,
			Optional:      true,
			ConflictsWith: []string{"vcs_repo.0.github_app_installation_id"},
		},
		"github_app_installation_id": {
			Type:          schema.TypeString,
			Optional:      true,
			ConflictsWith: []string{"vcs_repo.0.oauth_token_id"},
			AtLeastOneOf:  []string{"vcs_repo.0.oauth_token_id"},
		},
```